### PR TITLE
mello_2018_bromide_18562

### PR DESCRIPTION
**Pull request by markus-chem**
add lit folder

### CODE REVIEW

File: literature/mello_2018_bromide_18562/mello_2018_bromide_18562.bib [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@article{mello2018bromide,
+  title={Bromide Adsorption on Pt (111) over a Wide Range of pH: Cyclic Voltammetry and CO Displacement Experiments},
```
Comment (DunklesArchipel) [inline literature/mello_2018_bromide_18562/mello_2018_bromide_18562.bib:2]:
```suggestion
  title={Bromide Adsorption on Pt(111) over a Wide Range of pH: Cyclic Voltammetry and CO Displacement Experiments},
```

File: literature/mello_2018_bromide_18562/mello_2018_bromide_18562.bib [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@article{mello2018bromide,
```
Comment (DunklesArchipel) [inline literature/mello_2018_bromide_18562/mello_2018_bromide_18562.bib:1]:
```suggestion
@article{mello_2018_bromide_18562,
```

File: literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+          supplier:
+        - type:
+          model:
+          supplier:
+    electrolyte:
+        type: aq # aqueous = aq, non-aqueous=naq
+        components: # create as many components as necessary. A simple electrolyte consists of two components, i.e., H2SO4 and water
+            - name: HClO4 # can be trivia name, sum formula, etc
+              concentration:
+                  value:
```
Comment (DunklesArchipel) [inline literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml:39]: Is this not given in the manuscript.

**Review by DunklesArchipel — CHANGES_REQUESTED** [review]
The citation key was erroneous.
There might also be some missing entries in the component section.

File: literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    version: 1
+    instrumentation:
+        - type:
+          supplier: 
+        - type:
+          model:
+          supplier:
+        - type:
+          model:
+          supplier:
```
Comment (saraedum) [inline literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml:33]: What is the meaning of such a section?

File: literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+                  value: 
+                  unit:
+            - name: 
+              type: 
+              source:
+                  supplier:
+                  LOT: 
+              concentration:
+                  value:
+                  unit: #[M, mM, µM, g kg-1, ...]
```
Comment (saraedum) [inline literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml:68]: What is the meaning of this section?

File: literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml (hunk 3 above)
Comment (markus-chem) [reply literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml:39]: No, the concentration of HClO4, KClO4. Na2HPO4, NaH2PO4 and NaOH are not specified in the manuscript; only the KBr concentration. The pH is given in the paper (and listed in the yaml files).

File: literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml (hunk 4 above)
Comment (markus-chem) [reply literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml:33]: One could write here specifications of the used potentiostat, or other technical equipment. Often, they are not given in the publication.

File: literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml (hunk 5 above)
Comment (markus-chem) [reply literature/mello_2018_bromide_18562/mello_2018_bromide_18562_p1_1_ph1,2_black.yaml:68]: The components dissolved in the solvent.
In the digitization process, I fill out what is given in the paper and do not delete all empty entries.

**Review by DunklesArchipel — APPROVED** [review]

**Pull request merged**